README.md

```diff
@@ -42,4 +42,6 @@ docker-compose build
 docker-compose up
 ```
 
+Visit `localhost:8001` in a browser.
```

Comment:
This is not about the `linting`. Should this not be in another PR?

Reply:
Maybe in theory but the time it will take me to split the very small docs commits out is disproportionate and I'd rather not.